src/main/java/com/cisco/trex/stateless/model/ApiVers.java

```diff
@@ -0,0 +1,30 @@
+package com.cisco.trex.stateless.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+@JsonIgnoreProperties(ignoreUnknown = true) 
+public class ApiVers {
```

Comment:
I can see two models. They are ApiVers and ApiVersion and I'm confused. Please rename.